### DISCUSSION TITLE
Unable to download video

### DISCUSSION THREAD

**Discussion by xzicom** [aclap-dev/video-downloadhelper#743] [Help and Q&A]
### Your question

bug details below

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.strongwomancrushing.com/2019/03/28/maria-luiza-lifts-toys-and-destroys-a-refrigerator/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0000020463a0b800] Error when loading first segment 'https://d1j5fj1jn4udy7.cloudfront.net/output/hls/Marialuizarefrigerator/Marialuizarefrigerator.mp4_8500_00000.ts?Key-Pair-Id=K190RSOJGIW6J4&Signature=cgpehWkaB-Dwk4vkl7Ll~gF-hftXkKJiXa3~rhh3PIW6ATioAm-JcTQdNRGsW3A-e9jhX2hBwMiBvsvodcUcantS8Ijfk6CLQfJZybP0nf0HQqnReYB8Nb5elS23bj-~XbrkJCYsZEazvS4PPJQZDnYBGsKZKCVs1f1eryt1sUcIS-13Xb92af11taJBPbWv-2jmgm9Nw~71jue2zW0F7GD~a3VqWNW1foHZKwkz-YWgcaIQQkXIIVYk4q0PRWgHoQCvk5Xw3vNz7gE2RqDq1IWuPZvmzXmUGLMNWY-ZxlmKdRikSfqOz-sn4U-fqv8aSlz0ytmkgW4YhMUxL-Ohmw__&Expires=1716908597'\r\n[in#0 @ 0000020463a0b640] Error opening input: Invalid data found when processing input\r\nError opening input file https://www.strongwomancrushing.com/?stream_loader=https%3A%2F%2Fd1j5fj1jn4udy7.cloudfront.net%2Foutput%2Fhls%2FMarialuizarefrigerator.m3u8&pl=4&expire=1717066976&track=3401&signature=bdfeaf85c262459d0c24b1cf5ca1353f.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_51aff063-44ea-4d0f-80e2-5573f819220b",
    "downloadable_id": "downloadable_3447500297555624"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\mahes\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "yt_warning": false,
    "use_sidebar": false,
    "open_count_store": 210
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Unfortunately, that site won't let you play any content unless you buy a membership.  I'm not about to do that.

**Reply by xzicom** [reply]
Thanks for your reply. I have a membership but just unable to download the videos with VDH. I'm not sure if there's any DRM but having to use an alternative Bandicam to capture the content.
